spec/queries/patient_states/diabetes/cumulative_assigned_patients_query_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "rails_helper"
+
+describe PatientStates::Diabetes::CumulativeAssignedPatientsQuery do
+  around do |example|
+    with_reporting_time_zone { example.run }
+  end
+
+  let(:regions) { setup_district_with_facilities }
+  let(:period) { Period.current }
+
+  context "cumulative assigned patients" do
+    it "returns all the assigned patients in a facility as of the given period" do
+      facility_1_patients = create_list(:patient, 2, :diabetes, assigned_facility: regions[:facility_1])
+      facility_2_patients = create_list(:patient, 2, :diabetes, assigned_facility: regions[:facility_2])
+      refresh_views
```

Comment:
[flag, nonblocking?] The spec runs in the CI are slow (the last one took 1 hour), and adding a whole bunch of tests, that refreshes the views $spec_count number of times.

I am not sure how much this will help, but could try to only refresh the minimum required views to shave off time, like
```
RefreshReportingViews.call(views: [...])
```

The reason this review comment can be ignored for now IMO is that not all test runs seem that slow: https://simple.semaphoreci.com/branches/b6a0d51a-d06b-48d0-87f5-a8e252163358

But the older ones might have had lesser refreshes.